src/platform/middleware.ts

```diff
@@ -28,6 +28,12 @@ export const authorized: RequestHandler = (req, res, next) =>
 			res.sendStatus(401);
 		});
 
+export const identify: RequestHandler = (req, _res, next) =>
```

Comment:
What is this for?

Reply:
The `/device-types/v1` api can be called without any authentication, this adds a middleware that optionally authenticates the user.